app/models/event.rb

```diff
@@ -1,9 +1,14 @@
 # app/models/event.rb
 class Event < ApplicationRecord
+  NOTIFICATION_PERIOD = 1.day
```

Comment:
Unfamiliar with Rails, but this would be beneficial to be defined in a constants/config file for easier future changes.